src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -600,6 +613,16 @@ namespace cryptonote
       MERROR("Failed to parse reorg notify spec");
     }
 
+    try
+    {
+      if (!command_line::is_arg_defaulted(vm, arg_block_rate_notify))
+        m_block_rate_notify.reset(new tools::Notify(command_line::get_arg(vm, arg_block_rate_notify).c_str()));
+    }
+    catch (const std::exception &e)
+    {
+      MERROR("Failed to parse block rate notify spec");
```

Comment:
The exception may be due to parse failure or the file being missing, so something like this could be perhaps clearer:

```c++
       MERROR("Failed to set block rate notify spec: " << e.what());
```
This applies to all the other places where `Notify` is used (cryptonote_core and wallet2).